src/components/ContactDetails/ContactDetails.tsx

```diff
@@ -46,8 +46,12 @@ const ContactDetails: React.FC<ContactDetailsProps> = props => {
 
   return (
     <>
-      <Header type={'back'} onButtonClick={onBackClick} />
-      <PageContainer loading={loading} backGroundShape>
+      <Header
+        type={'back' as MenuTypes}
```

Comment:
Shouldn't we pass directly the enum here? And everywhere the header is imported
`MenuTypes.BACK`

Reply:
Yeah, in all other places it's like that. Good catch, I missed that 👁